src/main/scala/com/amazon/deequ/analyzers/Analyzer.scala

```diff
@@ -453,6 +471,13 @@ private[deequ] object Analyzers {
     conditionalSelection(col(selection), where)
   }
 
+  def conditionalSelection(selection: Column, where: Option[String], replaceWith: Any): Column = {
```

Comment:
I made `replaceWith` as type `Any` so different types can be replaced depending on the analyzers. Would it be better to leave it as Double for the length analyzers?